packages/voice/examples/recorder/package.json

```diff
@@ -14,15 +14,15 @@
 	"author": "Amish Shah <contact@shah.gg>",
 	"license": "MIT",
 	"dependencies": {
-		"@discordjs/opus": "^0.5.3",
 		"discord-api-types": "^0.30.0",
```

Comment:
Wrong version, not the same as the one d.js uses

Reply:
Should be 0.33.3, correct?

Reply:
Indeed!

https://github.com/discordjs/discord.js/blob/3638b4021ae3da8410c5626815b7d24f75160bcb/package.json#L52-L62